workflow/snakemake_rules/common.smk

```diff
@@ -207,6 +207,17 @@ def _get_max_date_for_frequencies(wildcards):
             datetime.date.today() - offset
         )
 
+def _get_narrow_bandwidth_for_wildcards(wildcards):
+    # check if builds.yaml contains frequencies:{build_name}:narrow_bandwidth
+    if wildcards.build_name in config["frequencies"] and 'narrow_bandwidth' in config["frequencies"][wildcards.build_name]:
+        return config["frequencies"][wildcards.build_name]["narrow_bandwidth"]
+    # check if parameters.yaml contains frequencies:default:narrow_bandwidth
+    elif "frequencies" in config and "narrow_bandwidth" in config["frequencies"]["default"]:
+        return config["frequencies"]["default"]["narrow_bandwidth"]
+    # else return augur frequencies default value
+    else:
+        return 0.0833
```

Comment:
_(non-blocking)_


```suggestion
```

This shouldn't be necessary since `defaults/parameters.yaml` already defines the default value and should always be used¹.

Non-blocking because `_get_min_date_for_frequencies` already has similar redundancy.

¹ under [expected usage](https://docs.nextstrain.org/projects/ncov/en/latest/guides/workflow-config-file.html#changing-parameters)